supportedBackends/google/pipelines/v2alpha1/src/test/resources/reference.conf

```diff
@@ -0,0 +1,2 @@
+demo.dos.docker-image = "somerepo/dos-downloader:tagged"
+demo.dos.command-template = "/path/to/some_executable before args ${cloudPath} middle args ${containerPath} ends args"
```

Comment:
And this runs the script along with passing the dos url and download location values?